rulette-core/src/main/java/com/github/kislayverma/rulette/core/ruleinput/value/defaults/InputDateValue.java

```diff
@@ -10,10 +10,15 @@
 class InputDateValue implements IInputValue<Date>, Serializable {
 
     private final Date value;
-    private static final DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyyMMdd");
-
-    public InputDateValue (String value) throws Exception {
-        this.value = value == null || value.isEmpty() ? null : formatter.parseDateTime(value).toDate();
+    private static final DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
Add comment saying this is the default MySql date format, which is the only supported format as of now